drafts/use-cases/1.1.security-considerations.md

```diff
@@ -10,23 +10,30 @@ There are several possibilities here. One of them would be to use a *@graph* not
 
 ```json
 {
+    "@context": "/api/context.jsonld",
     "@graph": [
         {
             "@id": "/api",
-            "operation": [
-                {
-                    "@type": "hydra:Operation",
-                    "title": "List events",
-                    "method": "GET",
-                    "returns": "hydra:Collection"
-                },
-                {
-                    "@type": "hydra:Operation",
-                    "title": "Create new event",
-                    "method": "POST",
-                    "expects": "schema:Event"
-                }
-            ]
+            "@type": "hydra:EntryPoint",
+            "#events": {
+                "@id": "/api/events",
+                "title": "List of events",
+                "@type": "hydra:Collection",
+                "operation": [
+                    {
+                       "@type": ["hydra:Operation", "schema:ViewAction"],
```

Comment:
Nah, I dunno. The spec defines `hydra:operation` to have range `hydra:Operation`. That's all you need really. Isn't that an answer to @asbjornu's initial comment?